packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.d.ts

```diff
@@ -0,0 +1,29 @@
+import { SFC, HTMLProps, ReactNode } from 'react';
+import { OneOf } from '../../typeUtils';
+
+export const ListVariant: {
+  grid: 'grid';
+  inline: 'inline';
+};
+
+export interface LoginPageProps extends HTMLProps<HTMLElement> {
+  children: ReactNode;
```

Comment:
all of these should be specified as optional `?`

Reply:
Looks like everything is required according to @andybraren 's comments

Reply:
As per your comment above, I'll fix this -  as loginTitle and form and the alt text should be the required props and the rest optional